src/database/sql/audio.py

```diff
@@ -210,9 +231,10 @@ def get_all(self, audio_id: str) -> str:
         """
 
         query = """
-                SELECT * 
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                SELECT *
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.

Reply:
Removed